### DISCUSSION TITLE
Is it possible Items not fitting into boxes to be packed individually?

### DISCUSSION THREAD

**Discussion by mastmaster** [dvdoug/BoxPacker#255] [General]
Hello!
Thank you and great work here.

I have testes successfully the test code
$packedBoxes = $packer->pack();

But when an item doesn't fit it throws an error and $packedBoxes is returned empty.

Is it possible to return $packedBoxes with the fitted items and not fitted items return as boxes from themselves dimensions?

thanks

**Comment by shakethatweight-simon** [comment]
@mastmaster  Have you looked at InfalliblePacker, this might be what you are looking for?

Look at the second example on this page: https://boxpacker.io/en/stable/too-large-items.html

**Comment by mastmaster** [comment]
I'm so sorry! I think this will help me a lot
